File: eRxWeb/SPA/Model/right-panel/failed-ref-req.model.ts
export class FailedRefReqModel {
    PatientName: string;
    PatientDOB: string;
    DrugDescription: string;
    Refills: string;
    PharmacyName: string;
    PharmacyAddress: string;
    PharmacyPhone: string;
    PharmacyFax: string;
}

export class DeniedRefReqMessages {
    RefReqErrors: FailedRefReqModel[];
    DeniedRefReqs: FailedRefReqModel[];   
}

File: eRxWeb/ServerModel/FailedRefReqModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace eRxWeb.ServerModel
{
    [Serializable]
    public class FailedRefReqModel
    {
        public string RequestID { get; set; }
        public string PatientName { get; set; }
        public string PatientDOB { get; set; }
        public string DrugDescription { get; set; }
        public string Refills { get; set; }
        public string PharmacyName { get; set; }
        public string PharmacyAddress { get; set; }
        public string PharmacyPhone { get; set; }
        public string PharmacyFax { get; set; }
    }
    [Serializable]
    public class DeniedRefReqMessages {
        public DeniedRefReqMessages()
        {
            this.DeniedRefReqs = new List<FailedRefReqModel>();
            this.RefReqErrors = new List<FailedRefReqModel>();
        }
        public IList<FailedRefReqModel> RefReqErrors { get; set; }
        public IList<FailedRefReqModel> DeniedRefReqs { get; set; }
    }
}